README.md

```diff
@@ -87,10 +63,34 @@ $ cog build -t my-colorization-model
 
 $ docker run -d -p 5000:5000 --gpus all my-colorization-model
 
-$ curl http://localhost:5000/predict -X POST -F input=@image.png
+$ curl http://localhost:5000/predictions -X POST \
+   --data '{"input": "https://.../input.jpg"}'
 ```
 
-That's it! Your model will now run forever in this reproducible Docker environment.
+<!-- In development, you can also run arbitrary commands inside the Docker environment:
+
+```
+$ cog run python train.py
+...
+``` -->
+
+<!-- TODO: this doesn't work yet (needs ports etc)
+Or, spin up a Jupyter notebook:
+
+```
+$ cog run jupyter notebook
+```
+-->
+
+## Deploying models to production
+
+Cog does a number of things out of the box to help you deploy models to production:
+
+- **Standard interface.** Put models inside Cog containers, and they'll run anywhere that runs Cog containers.
```

Comment:
`they'll run anywhere that runs Cog containers.` --- Should this be **Docker** instead of **Cog**

Reply:
I mean, technically correct, but yes that is a more useful statement haha.